packages/cyberstorm/src/components/componentStyles/CustomSelectItem.module.css

```diff
@@ -0,0 +1,50 @@
+.root {
+  display: flex;
+  flex-direction: row;
+
+  gap: var(--gap-m);
+  min-width: 10rem;
+  padding: var(--padding-3xs) var(--padding-xs);
+  border-radius: var(--border-radius-3xs);
+  overflow: hidden;
+  color: var(--text-color);
+
+  font: var(--font-body);
+  font-size: var(--text-s);
+  background-color: var(--bg-color);
```

Comment:
I don't think this should have `background-color` set by default as it should let the `background-color` of the parent container through. It also creates a visual z-index problem with the styling `reset.css` applies to `:focus-visible` -state.